src/components/legend/style_utils.ts

```diff
@@ -68,18 +69,22 @@ export function getLegendListStyle(
   return {
     paddingTop,
     paddingBottom,
+    ...(floating && {
+      display: 'grid',
+      gridTemplateColumns: `repeat(${clamp(floatingColumns ?? 1, 1, totalItems)}, 1fr`,
```

Comment:
`1fr` will cause the largest string to control the width for all columns like this....

![image](https://user-images.githubusercontent.com/19007109/118987185-a52cf100-b945-11eb-86d9-3e5c524e7eb6.png)

If we use `auto` instead of `1fr` then each column width will be independent. Not sure which you think is desirable but I like the independent widths.

![image](https://user-images.githubusercontent.com/19007109/118987129-98a89880-b945-11eb-9e39-24c9748d0ee1.png)

```suggestion
      gridTemplateColumns: `repeat(${clamp(floatingColumns ?? 1, 1, totalItems)}, auto`,
```

Reply:
fixed in 3b1cfc4